test/ui/DateUtilsTest.ts

```diff
@@ -0,0 +1,59 @@
+import { DateUtils } from '../../src/utils/DateUtils';
+import { IDateToStringOptions } from '../../src/utils/DateUtils';
+import * as moment from 'moment';
+import { l } from '../../src/strings/Strings';
+
+export function DateUtilsTest() {
+
+  describe('DateUtils', () => {
```

Comment:
There's really not a lot of test.

I understand it's kinda covered by the template helper class, but are you sure you are checking correctly for corner cases ?

For example, what happens if we set a locale that does not exists ?

A test for the /yyyy/ to upper case  ? 

A test for weird formats in `convertToStandardDate` ? Does it crash in any case, or does it correctly return null for all kind of garbage we throw in ?

etc...